src/runtime_src/ert/scheduler/scheduler_v30.cpp

```diff
@@ -1081,7 +1081,8 @@ scheduler_v30_loop()
           auto cqvalue = read_reg(slot.slot_addr);
 
           if (cqvalue & (AP_START|AP_CONTINUE)) {
-            write_reg(slot.slot_addr,0x0); // clear
+            while(read_reg(slot.slot_addr))
+              write_reg(slot.slot_addr,0x0); // clear
```

Comment:
why do we still need this change? shouldn't the one line change below just do it?